src/views/catalog/templatescatalog/components/TemplatesCatalogTile.tsx

```diff
@@ -31,7 +31,7 @@ export const TemplateTile: React.FC<TemplateTileProps> = React.memo(({ template,
   const workload = getTemplateWorkload(template);
   const displayName = getTemplateName(template);
   const bootSourceType = getTemplateBootSourceType(template)?.type;
-  const { memory, cpuCount } = getFlavorData(template);
+  const { memory, cpuCount } = getTemplateFlavorData(template);
```

Comment:
looks like you can have more information about the CPU:
http://kubevirt.io/api-reference/main/definitions.html#_v1_cpu

removing flavor allows us to show more correct data here, e.g. cores and threads instead of the flavor code number
not sure showing `const cpu = core * thread * socket;` makes sense

@stu-gott hi, does it make sense for the UI to show "CPU: < core * thread * socket >" , should we move to something more verbose, like "Cores: <core> Threads: <thread>, Sockets: <socket>" , should we show only `Cores` and ignore threads and sockets ?